Sources/Private/CoreAnimation/CompatibilityTracker.swift

```diff
@@ -126,3 +126,12 @@ extension LayerAnimationContext: CompatibilityTrackerProviding {
     currentKeypath.fullPath
   }
 }
+
+extension Array where Element == CompatibilityIssue {
+  var compatibilityMessage: String {
+    let lines = ["Encountered Core Animation compatibility issues while setting up animation:"]
```

Comment:
Probably be worth including details on how to recover from this error in both types of animation configuration